components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -265,6 +289,26 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             .send(ApplyTask::CommittedEntries(apply));
     }
 
+    #[inline]
+    fn report_store_time_duration<T>(
+        &mut self,
+        ctx: &mut StoreContext<EK, ER, T>,
+        entry_and_proposals: &mut [(Entry, Vec<CmdResChannel>)],
+    ) {
```

Comment:
missing checks like in `report_batch_wait_duration`

Reply:
It's on purpose. v1 doesn't check it either.